walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -28,28 +28,45 @@
     public int compare(Event a, Event b) {
       return Long.compare(a.getWhen().start(), b.getWhen().start());
     }};
-
-  public Collection<TimeRange> query(Collection<Event> eventCollection, MeetingRequest request) { 
+    
+  public Collection<TimeRange> query(Collection<Event> eventCollection, MeetingRequest request) {
+    Collection<String> attendees = new ArrayList<String>(); 
```

Comment:
Ah, heres the collection.

Yeah, I said before to use generic types, but `Collection` is too generic, lol. List is probably what you want.